README.md

```diff
@@ -22,37 +26,43 @@ An example application implementing an automated phone line using Twilio and Exp
    $ mongod
    ```
 
-2. Clone this repository and `cd` into its directory:
+1. Clone this repository and `cd` into its directory:
    ```
```

Comment:
We'd like to make the READMEs more visually appealing by highlighting code blocks. Would you mind using **```bash**?